### PR TITLE
sdk integration test

### PR DESCRIPTION
**Pull request by dboreham**
WIP: 

This allows creation of two containers : `cerc-io/laconicd:local-test` , built from the current local tree and `cerc-io/laconic-sdk-tester:local-test` which contains the laconic-sdk project cloned from git in the container build process.

Handy scripts `build-laconicd-container.sh` and `build-sdk-test-container.sh` for building those containers.

Then you can run `docker compose up -d` to spin up laconicd and the test runner container networked together.

And then you run `./run-tests.sh` to run the laconic-sdk test suite.

Alas at present all tests fail with:

```
    thrown: Array [
      Object {
        "message": "rpc error: code = NotFound desc = account ethm1zhhartn9s7xv03dze45y93nedusyhq58magakl not found",
        "path": Array [
          "getAccounts",
        ],
      },
    ]
```

Usually this is a symptom of the chain id being wrong. However I've corrected it (I think...) and the errors persis.
So a second cycle of debugging the failures is required.

### CODE REVIEW

**Comment by 0xmuralik** [comment]
There must be some problem with scripts
After running docker compose up I tried to run a cli qurey to the chain and it fails
Ex: `laconicd q bank balances ethm1m7cnnpqk24r42h7dt0t4d7l2qh0uw46muv4qtd
Error: post failed: Post "http://localhost:26657": dial tcp 127.0.0.1:26657: connect: connection refused`
Either the chain didn't start or the ports must be different

**Comment by dboreham** [comment]
> After running docker compose up I tried to run a cli qurey to the chain and it fails Ex: `laconicd q bank balances ethm1m7cnnpqk24r42h7dt0t4d7l2qh0uw46muv4qtd Error: post failed: Post "http://localhost:26657": dial tcp 127.0.0.1:26657: connect: connection refused` The ports might be different.

Hmm. This works for me:

```
$ docker compose exec laconicd laconicd q bank balances ethm1m7cnnpqk24r42h7dt0t4d7l2qh0uw46muv4qtd
balances: []
pagination:
  next_key: null
  total: "0"
```

File: init.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 KEY="mykey"
-CHAINID="ethermint_9000-1"
+CHAINID="laconic_9000-1"
```
Comment (dboreham) [inline init.sh:4]: This is the correct chain id.

**Comment by zramsay** [comment]
not sure if this error is still an issue: `Error: post failed: Post "http://localhost:26657": dial tcp 127.0.0.1:26657: connect: connection refused` but it looked instantly familiar and this answer has worked for me in the past: [https://stackoverflow.com/a/61345315](https://stackoverflow.com/a/61345315)

**Pull request merged**